printrun/gcview.py

```diff
@@ -64,13 +64,13 @@ def set_gcview_params(self, path_width, path_height):
     for obj in self.objects[1:]:
         if isinstance(obj.model, actors.GcodeModel):
             obj.model.set_path_size(self.path_halfwidth, self.path_halfheight)
-            has_changed = True
+            has_changed = True                      
```

Comment:
Spurious spaces where introduced here. Could you delete them?

Reply:
Ups. Yes, this is corrected with the next update.